src/os_crypto/sha1/sha1_op.c

```diff
@@ -143,7 +149,7 @@ int OS_SHA1_File_Nbytes_with_fp_check(const char * fname, SHA_CTX * c, os_sha1 o
     memset(output, 0, sizeof(os_sha1));
     buf[OS_MAXSTR - 1] = '\0';
 
-    SHA1_Init(c);
+    EVP_DigestInit(*c, EVP_sha1());
```

Comment:
What about `EVP_MD_CTX_new`? If not necessary, please explain in the function header that the context won't be created / freed by this function.

Reply:
Comment added to the function header.